main.js

```
@@ -48,3 +50,140 @@ navbarBtn.addEventListener('click', () => {
+
+    const liText = document.createTextNode(item);
+    li.appendChild(liText);
+    fragment.appendChild(li);
+  });
+  return fragment;
+};
+
+const createProject = (project) => {
+  const projectDiv = document.createElement('div');
+  projectDiv.className = 'project';
+  const cardDiv = document.createElement('div');
+  cardDiv.className = 'card';
+  const cardImageDiv = document.createElement('div');
+  cardImageDiv.className = 'card-image';
+  const img = document.createElement('img');
+  img.src = project.image;
+  cardImageDiv.appendChild(img);
+  cardDiv.appendChild(cardImageDiv);
+  const cardBodyDiv = document.createElement('div');
+  cardBodyDiv.className = 'card-body';
+  const BodyTitleDiv = document.createElement('div');
+  BodyTitleDiv.className = 'card-title';
+  const h2 = document.createElement('h2');
+  const h2Text = document.createTextNode(project.name);
+  h2.appendChild(h2Text);
+  BodyTitleDiv.appendChild(h2);
+  cardBodyDiv.appendChild(BodyTitleDiv);
+  const ClientInfoDiv = document.createElement('div');
+  ClientInfoDiv.className = 'card-client-info';
+  const ul = document.createElement('ul');
+  const itemlist = ['CANOPY', 'Back End Dev', '2015'];
+  const lists = repeatList(itemlist, 'info');
+  ul.appendChild(lists);
+  ClientInfoDiv.appendChild(ul);
+  cardBodyDiv.appendChild(ClientInfoDiv);
+
+  const BodyPragrphDiv = document.createElement('div');
+  BodyPragrphDiv.className = 'card-pragrph';
+  const p = document.createElement('p');
+  const pText = document.createTextNode(project.description);
+  p.appendChild(pText);
+  BodyPragrphDiv.appendChild(p);
+  cardBodyDiv.appendChild(BodyPragrphDiv);
+
+  const CardTechDiv = document.createElement('div');
+  CardTechDiv.className = 'card-tech';
+  const ulTech = document.createElement('ul');
+  const listTech = project.technologies;
+  const listTechs = repeatList(listTech);
+  ulTech.appendChild(listTechs);
+  CardTechDiv.appendChild(ulTech);
+  cardBodyDiv.appendChild(CardTechDiv);
```

[OPTIONAL] Nice list 👍🏽 but how about some error checking for invalid data: You could also add checks to validate the data types of certain fields, such as `technologies`. For example, you could ensure that `technologies` is an array of strings. If the data is invalid, you could throw an error or log a warning. ❗️